src/includes/Exports-Include.hs

```diff
@@ -49,6 +56,25 @@
   , setCursorPositionCode
 
     -- * Saving, restoring and reporting cursor position
+    -- | These code sequences are not part of ECMA-48 standard; they are popular,
+    -- but non-portable extensions. E. g., Terminal.app on MacOS
+    -- <https://stackoverflow.com/questions/25879183 does not support them>.
```

Comment:
Referring to my 'document the exceptions' comment above, as Terminal is the native terminal software on macOS, and macOS is clearly an important operating system, I do think it is worth warning users of limitations of versions of that terminal software that are supported by Apple (and not obsolete). However, this is the practical difficulty for this library and its documentation: keeping on top of the universe of terminal software, especially when its versions and their capabilities are not well documented (which I think is the case for macOS's Terminal).